libp2p/crypto/curve25519.nim

```diff
@@ -16,6 +16,7 @@
 # RFC @ https://tools.ietf.org/html/rfc7748
 
```

Comment:
Please use common API names for all procedures here, it will be much more easier to maintain and review code which is compatible between modules. You can see examples of API:
* https://github.com/status-im/nim-libp2p/blob/master/libp2p/crypto/ed25519/ed25519.nim
* https://github.com/status-im/nim-libp2p/blob/master/libp2p/crypto/ecnist.nim

Because Curve25519 implementation is BearSSL implementation it would be much more nicer to use BearSSL CSPRNG implementation to generate keys.

Reply:
To which proc you refer ? Cos they are a little bit different, like there is no definition of private or public key, public is just a key multiplied by `9.byte` literally internally